lib/src/gaskets/spi/spi_sub.dart

```diff
@@ -48,6 +54,18 @@ class SpiSub extends Module {
 
     addOutput('done');
 
+    // Counter to track of the number of bits shifted out.
+    final count = Counter.simple(
+        clk: intf.sclk,
+        enable: ~intf.csb,
+        reset: reset ?? Const(0, width: 1),
```

Comment:
Hmm, if there's no reset, then maybe this whole counter shouldn't be here, actually.  That would mean the `done` signal is dependent on a `reset` being provided.